docs/clustered.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Backup & Restore Clustered Percona XtraDB Database | Stash
+description: Backup Clustered Percona XtraDB Database database using Stash
+menu:
+  product_stash_{{ .version }}:
+    identifier: percona-xtradb-clustered-guide-{{ .subproject_version }}
+    name: Backup & Restore Clustered Percona XtraDB
+    parent: stash-percona-xtradb-guides-{{ .subproject_version }}
+    weight: 10
+product_name: stash
+menu_name: product_stash_{{ .version }}
+section_menu_id: stash-addons
+---
+
+**Deploy Database:**
+
+```bash
+$ kubectl apply -f ./docs/examples/clustered/backup/sample-xtradb-cluster.yaml
+perconaxtradb.kubedb.com/sample-xtradb-cluster created
+```
+
+**Create Secret:**
+```bash
+$ kubectl create secret generic -n demo gcs-secret \
+                --from-file=./RESTIC_PASSWORD \
+                --from-file=./GOOGLE_PROJECT_ID \
+                --from-file=./GOOGLE_SERVICE_ACCOUNT_JSON_KEY
+```
+
+**Create Repository:**
+
+```bash
+$ kubectl apply -f ./docs/examples/clustered/backup/repository.yaml
+repositories.stash.appscode.com/gcs-repo-xtradb-cluster created
+```
+
+**Install Chart:**
+
+```console
+$ helm template chart/stash-percona-xtradb --name=percona-xtradb-5.7  --set docker.registry=appscodeci --set docker.tag=refactor_linux_amd64 | kubectl apply -f -
```

Comment:
I don't think we should show command using ci registry.

Reply:
This is a template document. Shudipta will complete this doc. Installation part will not be here. I have just noted the steps for Shudipta.